packages/drivers/bun-pg/src/BunPgDriver.ts

```diff
@@ -124,7 +125,7 @@ function batchInsert(txn: TransactionSQL, op: InsertOp): Promise<unknown> {
 async function batchUpdate(txn: TransactionSQL, op: UpdateOp): Promise<void> {
   const { tableName, columns, rows, updatedAt } = op;
 
-  const cte = buildValuesCte("data", columns, rows);
+  const cte = buildValuesCte("data", columns, rows, rows.flat());
```

Comment:
Previously the bindings were done separately from the `buildValuesCte`, but now passing them into, b/c the `buildValuesCte` doesn't just make a string (as before) it makes the `ParsedCteClause`